try:
    number1=5
    number2=6
    print(number1*number2)


except Exception as e:
    print(e)